cli/src/java/org/apache/hadoop/hive/cli/CliDriver.java

```diff
@@ -250,12 +250,13 @@ CommandProcessorResponse processLocalCmd(String cmd, CommandProcessor proc, CliS
         }
 
         // Set HDFS CallerContext to queryId and reset back to sessionId after the query is done
-        ShimLoader.getHadoopShims().setHadoopQueryContext(qp.getQueryState().getQueryId());
+        ShimLoader.getHadoopShims()
+            .setHadoopQueryContext(qp.getQueryState().getQueryId() + "_User:" + ss.getUserName());
```

Comment:
can we extract "_User:" literal into a constant or have a pattern for String.format() ?

Reply:
Changed